File: Backend.Domain/Entities/Cliente.cs
namespace Backend.Domain.Entities
{
    public class Cliente : Entity
    {
        public string Nome { get; set; } = default!;
        public string Email { get; set; } = default!;
        public string Telefone { get; set; } = default!;
        public bool Ativo { get; set; } = true;
    }
}

File: Frontend/src/app/core/models/cliente.ts
import { dateToString } from "../../../utils/dateToString";

export class Cliente {
    public id!: number;
    public nome!: string;
    public email!: string;
    public telefone!: string;
    public criadoEm: Date = new Date();
    public ativo: boolean = false;

    constructor() { }

    get ativoFormatado(): string {
        return this.ativo ? 'Sim' : 'Não';
    }

    get criadoEmFormato(): string {
        const date = dateToString(this.criadoEm);
        return date;
    }
}
